### ISSUE TITLE
Custom Fields Values Not Being Passed

### ISSUE THREAD

**Issue by KnightAR** [hellosign/hellosign-php-sdk#121]
I'm having a difficult time passing pre-filled values using a template in a EmbededSignatureRequest flow. When I request with the exact field names I set in the template, it only returns back a SigntureRequest object with the custom fields values blank. 

For example: 

```
$request = new \HelloSign\TemplateSignatureRequest;
...
$request->setCustomFieldValue('ApplicantName1', $this->application->ContactName);

$embedded_request = new \HelloSign\EmbeddedSignatureRequest($request, config('hellosign.client_id'));

$response = $client->createEmbeddedSignatureRequest($embedded_request);
```

Even tho I have the ApplicantName1 field on the template, the value always returns NULL within the returned object and while signing the document.

I've confirmed that the JSON is being passed correctly to the REST API.

`
"custom_fields" => '[{"name":"ApplicantName1","value":"Test Person","editor":null,"required":null}]',
`

Any help would be appricated. I have a call with a HelloSign CSM tomarrow, so hopefully want to see if I'm doing something wrong before I discuss the issue with them.

**Comment by mikerhino** [comment]
@KnightAR Did you ever get this working? I'm running into the same issue. I can verify that the custom field data is properly formatted in the EmbeddedSignatureRequest payload but the return from createEmbeddedSignatureRequest always has blank value fields for the custom fields.

**Comment by mikerhino** [comment]
For people viewing this in the future, it looks like our issue was actually with the template setup. Per this FAQ document from HelloSign:

https://faq.hellosign.com/hc/en-us/articles/360003334591-Reason-why-custom-fields-aren-t-populating

We had to set those fields to filled by sender. Once we did that, we were able to set those values through the SDK no problem.

**Comment by KnightAR** [comment]
@mikerhino Yes, correct. I ended up getting it working. Also FYI, if you have fields that are named the same, only one will be filled out. We had to create seperate fields for each field even tho the value was idential. Just little things.

**Comment by mikerhino** [comment]
@KnightAR Oh hey thats a great tip! We've got ours numbered (ie: FullName1) but I could see how that might trip up someone in the future. Thanks for the reply!
